### DISCUSSION TITLE
Implementation preference on issue #567

### DISCUSSION THREAD

**Discussion by hsergiu** [Project-Books/book-project#658] [Q&A] [answered]
There are two different components used for the e-mail and password fields. One is a textfield and the other one is a form control. Should I use a textfield for the password too since it is easier to change the props and to edit it in general?

**Comment by knjk04** [comment, marked as answer]
The reason for using different components is we want a visibility toggle button on the password fields. I don't think we can get that using `TextField`s alone. I think it's best to leave the password field as an `OutlinedInput` for now
